doc/contribute/project_roles.rst

```diff
@@ -113,7 +129,23 @@ Release Engineering Team
 ########################
 
 A team of active Maintainers involved in multiple areas.
-The team is appointed by the TSC and reports directly into the TSC.
+
+* The Release Engineering team contains the group of individuals eligible
+  to be appointed as Release Managers for Zephyr RTOS releases.
+* Release Engineering team members are expected to be available,
```

Comment:
> Release Engineering team members are expected to be available,
>  from time to time, for serving as Release Managers.

How about:
* Members of the release engineering team are expected to fill the release engineering manager role based on a defined cadence and selection process.

Reply:
Yes, sure; it is more accurate, actually, based on how we discussed it.

Reply:
Updated this now.